packages/vsphere-csi/charts/values.yaml

```diff
@@ -0,0 +1,70 @@
+vCenter:
+  host: ""
+  port: 443
+  insecureFlag: true
```

Comment:
Should this be "1" or true ? @PennyScissors

Reply:
Switching to 1